test/functional/tests/io_class/test_io_classification.py

```diff
@@ -70,37 +70,40 @@ def test_ioclass_lba():
                     .count(dd_count)
                     .block_size(dd_size)
                     .seek(lba)
+                    .oflag("direct")
             )
             dd.run()
-            sync()
             dirty_count += 1
 
             dirty = cache.get_io_class_statistics(io_class_id=ioclass_id).usage_stats.dirty
             if dirty.get_value(Unit.Blocks4096) != dirty_count:
                 TestRun.LOGGER.error(f"LBA {lba} not cached")
 
-    with TestRun.step("Flush cache."):
-        cache.flush_cache()
-
     with TestRun.step("Run IO and check if lba outside of defined range are not cached."):
-        TestRun.LOGGER.info(f"Writing to random sectors outside of cached range.")
-        for i in range(iterations):
```

Comment:
iterations variable is unused now, remove it

Reply:
Fixed